specification/communication/data-plane/PhoneNumbers/preview/2021-10-25-preview/phonenumbers.json

```diff
@@ -0,0 +1,1639 @@
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink",
+          "itemName": "programBriefs"
+        },
+        "x-ms-examples": {
+          "List US Program Briefs": {
+            "$ref": "./examples/GetUSProgramBriefsAsync.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "PhoneNumberCapabilities": {
+      "description": "Capabilities of a phone number.",
+      "required": [
+        "calling",
+        "sms"
+      ],
+      "type": "object",
+      "properties": {
+        "calling": {
+          "description": "Capability value for calling.",
+          "enum": [
+            "none",
+            "inbound",
+            "outbound",
+            "inbound+outbound"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "PhoneNumberCapabilityType",
+            "modelAsString": true
+          }
+        },
+        "sms": {
+          "description": "Capability value for SMS.",
+          "enum": [
+            "none",
+            "inbound",
+            "outbound",
+            "inbound+outbound"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "PhoneNumberCapabilityType",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "PhoneNumberSearchRequest": {
+      "description": "Represents a phone number search request to find phone numbers. Found phone numbers are temporarily held for a following purchase.",
+      "required": [
+        "assignmentType",
+        "capabilities",
+        "phoneNumberType"
+      ],
+      "type": "object",
+      "properties": {
+        "phoneNumberType": {
+          "description": "The type of phone numbers to search for, e.g. geographic, or tollFree.",
+          "enum": [
+            "geographic",
+            "tollFree"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "PhoneNumberType",
+            "modelAsString": true
+          }
+        },
+        "assignmentType": {
+          "description": "The assignment type of the phone numbers to search for. A phone number can be assigned to a person, or to an application.",
+          "enum": [
+            "person",
+            "application"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "PhoneNumberAssignmentType",
+            "modelAsString": true
+          }
+        },
+        "capabilities": {
+          "$ref": "#/definitions/PhoneNumberCapabilities"
+        },
+        "areaCode": {
+          "description": "The area code of the desired phone number, e.g. 425.",
+          "type": "string"
+        },
+        "quantity": {
+          "format": "int32",
+          "description": "The quantity of desired phone numbers. The default value is 1.",
+          "default": 1,
+          "maximum": 2147483647,
```

Comment:
I'm guessing this isn't a meaningful value to the service.  I'd just leave it off because it's already implied by the format `int32`.